app/views/site/_study_visualize.html.erb

```diff
@@ -45,6 +45,7 @@
 <script type="text/javascript" nonce="<%= content_security_policy_script_nonce %>">
   window.SCP.studyAccession = '<%= @study.accession %>';
 
-  window.SCP.exploreDefault();
+  //window.SCP.exploreDefault();
+  window.SCP.renderExploreView($('#study-visualize')[0], window.SCP.studyAccession)
```

Comment:
Style nit: I feel that calling `$('#study-visualize')[0]` defeats the purpose of using jQuery altogether.  While more verbose, `document.getElementById('study-visualize')` returns the actual DOM element not wrapped in a jQuery function.  There's probably an easier way to do it as well.